rust/utils/run-locally/src/main.rs

```diff
@@ -147,17 +147,12 @@ fn main() -> ExitCode {
         "HYP_BASE_CHAINS_TEST2_CONNECTION_URLS" => "http://127.0.0.1:8545,http://127.0.0.1:8545,http://127.0.0.1:8545",
         "HYP_BASE_CHAINS_TEST2_CONNECTION_TYPE" => "httpQuorum",
         "HYP_BASE_CHAINS_TEST3_CONNECTION_URL" => "http://127.0.0.1:8545",
-        "HYP_BASE_CHAINS_TEST3_CONNECTION_TYPE" => "http",
```

Comment:
are these changes because they will result in errors now, or just that they now are the default?

Reply:
They are now default.